### ISSUE TITLE
Display sonar result of the generated code 

### ISSUE THREAD

**Issue by nicolas63** [jhipster/jhipster-dotnetcore#1418]
##### **Overview of the feature request**

We scan the code of generated sample app only after the merge. 

##### **Motivation for or Use Case**

Scan the generated code during the PR 

##### **Related issues or PR**

- [x] Checking this box is mandatory (this is just to show you read everything)

**Comment by A-5ingh** [comment]
@nicolas63  Can you provide more details on this?

**Comment by nicolas63** [comment]
Yes, today we have this label in the README: https://github.com/jhipster/jhipster-dotnetcore/blob/main/README.md#analysis-of-the-sample-project. It is updated only when we merge into the main branch. The analysis is done by SonarCloud. The goal of this task is to assist during the code review by providing the Sonar analysis on the PR. For the main generator, it is done by this script (https://github.com/jhipster/generator-jhipster/blob/main/test-integration/scripts/25-sonar-analyze.sh) with a local SonarQube instance. We can do the same thing and maybe comment on the PR with the result. I tried using SonarCloud on the PR, but when I tried, the secret variable was not shared in the PR context (maybe it's possible today). If you need more information or help don't hesitate

**Comment by nicolas63** [comment]
an other script is also here in jhlite (https://github.com/jhipster/jhipster-lite/blob/main/tests-ci/sonar.sh)
